packages/auth/src/AuthProvider.tsx

```diff
@@ -124,11 +124,17 @@ export class AuthProvider extends React.Component<
       `${global.__REDWOOD__API_PROXY_PATH}/graphql`,
       {
         method: 'POST',
-        headers: {
-          'content-type': 'application/json',
-          'auth-provider': this.rwClient.type,
-          authorization: `Bearer ${token}`,
-        },
+        headers:
+          token === null
+            ? {
+                'content-type': 'application/json',
+                'auth-provider': this.rwClient.type,
+              }
+            : {
+                'content-type': 'application/json',
+                'auth-provider': this.rwClient.type,
+                authorization: `Bearer ${token}`,
+              },
```

Comment:
The token will be `null` in case the provider is using `httpOnly` cookies. In that case, we do not need the `authorization` header